### ISSUE TITLE
Add --plot-to-board to plot command

### ISSUE THREAD

**Issue by Ana06** [openSUSE/trollolo#134]
This option would be also useful in this command. This will send the plotted burndown chart to the first card of the `Done` column.

**Comment by ankitkataria** [comment]
can i claim this issue too?

**Comment by Ana06** [comment]
@ankitkataria what about finishing #138 before starting with this one? 😉

**Comment by Ronaq13** [comment]
Can I take this issue

**Comment by Ana06** [comment]
@Ronaq13 sure! :+1:

**Comment by Ronaq13** [comment]
Started working on it.

**Comment by Ronaq13** [comment]
@Ana06 where is the contribution.md for this project? How can I set up this project?

**Comment by Ronaq13** [comment]
@Ana06 I was trying my hands on it and have two doubts.

1) When I tried to create a plot by `trollolo plot 01` but it gave me the following error:
`create_burndown.py", line 39
    print args
             ^
SyntaxError: Missing parentheses in call to 'print'. Did you mean print(int args)?`
Rest of the integration I have done and I am able to run `trollolo get-cards --board-id=MyTrelloWelcomeBoardID`

When I fixed this by doing `print(args)`, then it gave me the following error:

`burndowndata.py", line 84, in getSprintData
    if day.has_key("fast_lane"):
AttributeError: 'dict' object has no attribute 'has_key'`

2) Whenever I make changes do I need to re-build and re-install the gem everytime, to see changes?

**Comment by Ana06** [comment]
@Ronaq13 can you send a PR with the changes? it is really difficult to find out what you exactly did without seeing the code :wink:

**Comment by Ronaq13** [comment]
I haven't made any changes regarding this issue. I just add parentheses
around `args` in print statement. I was trying to understand, what exactly
trollolo does, so I was just trying out it features.

**Comment by Ana06** [comment]
Sorry @Ronaq13 , I got you wrong. If you didn't change anything, and you find something that doesn't work, it may be a bug and then it would be better to open a separate issue :wink:

**Comment by Ronaq13** [comment]
I encounter an issue #172 while solving this issue.

**Comment by Ronaq13** [comment]
It was a python vesion conflict. back on this issue

**Comment by Ana06** [comment]
@Ronaq13 

> Whenever I make changes do I need to re-build and re-install the gem every time, to see changes?

As I told you by email, yes. You can also write a test.

The rest, I'll answer in the PR

**Comment by Ana06** [comment]
There is some work done in https://github.com/openSUSE/trollolo/pull/184
